packages/website/src/docs/standard-library/built-in-types.md

```diff
@@ -0,0 +1,53 @@
+---
+id: built-in-types
+title: Built-in types
+---
+
+# Built-in Types
+
+Cadl Standard Library provide some built-in types that can be used to build more complex types.
+
+See [type relations]({%doc "type-relations"%}) for details on how those types are structred.
```

Comment:
I can't quite parse the meaning of this reference text - maybe something like "Built in types are related to each other according to the rules described in type relations".